src/keys.ts

```diff
@@ -123,20 +132,27 @@ export async function getPaymentKeyInfo(network: CLINetworkAdapter, mnemonic : s
  *    .privateKey (string) the hex private key
  *    .address (string) the address of the private key
  */
-export async function getStacksWalletKeyInfo(network: CLINetworkAdapter, mnemonic : string): Promise<PaymentKeyInfoType> {
-  const seed = await bip39.mnemonicToSeed(mnemonic)
-  const master = bip32.fromSeed(seed)
-  const child = master.derivePath(`m/44'/5757'/0'/0/0`)     // taken from stacks-wallet. See https://github.com/blockstack/stacks-wallet
-  const ecPair = bitcoin.ECPair.fromPrivateKey(child.privateKey)
-  const privkey = blockstack.ecPairToHexString(ecPair)
+export async function getStacksWalletKeyInfo(network: CLINetworkAdapter, mnemonic : string): Promise<StacksKeyInfoType> {
+  const seed = await bip39.mnemonicToSeed(mnemonic);
+  const master = bip32.fromSeed(seed);
+  const child = master.derivePath('m/44\'/5757\'/0\'/0/0');     // taken from stacks-wallet. See https://github.com/blockstack/stacks-wallet
```

Comment:
```suggestion
  const child = master.derivePath(`m/44'/5757'/0'/0/0`);     // taken from stacks-wallet. See https://github.com/blockstack/stacks-wallet
```
autoformatted to the escaped string?

Some of this code is in the keychain library, too.